utils/zerovec/src/zerovec/mod.rs

```diff
@@ -130,6 +130,7 @@ struct EyepatchHackVector<U> {
     /// This pointer is *always* valid, the reason it is represented as a raw pointer
     /// is that it may logically represent an `&[T::ULE]` or the ptr,len of a `Vec<T::ULE>`
     buf: NonNull<[U]>,
+    #[cfg(feature = "alloc")]
```

Comment:
not: explicitly document behavior when not present

Reply:
hm? there's no behaviour change, this is just always 0 in no-alloc